core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -5106,30 +5106,6 @@ protected function listView()
 					{
 						$field = $GLOBALS['TL_DCA'][$this->strTable]['list']['label']['fields'][$j] ?? null;
 
-						if (isset($GLOBALS['TL_DCA'][$this->strTable]['fields'][$field]['foreignKey']))
-						{
-							if ($arg)
-							{
-								$key = explode('.', $GLOBALS['TL_DCA'][$this->strTable]['fields'][$field]['foreignKey'], 2);
-
-								$reference = $this->Database
-									->prepare("SELECT " . Database::quoteIdentifier($key[1]) . " AS value FROM " . $key[0] . " WHERE id=?")
-									->limit(1)
-									->execute($arg);
-
-								if ($reference->numRows)
-								{
-									$arg = $reference->value;
-								}
-							}
-
-							$value = $arg ?: '-';
-						}
-						else
-						{
-							$value = (string) $arg !== '' ? $arg : '-';
-						}
-
```

Comment:
Removing this causes an error on my system:

<img width="1043" alt="" src="https://github.com/contao/contao/assets/1192057/d717f6e9-3af0-48db-9237-0277833de324">

Reply:
Fixed in 5265c250436d5b54e35fd4166a2e7821a1607742